ValetTests/ValetTests.m

```diff
@@ -90,15 +91,20 @@ - (void)setUp;
     self.string = @"bar";
     self.secondaryString = @"bar2";
     self.additionalValets = [NSMutableArray new];
+    self.manualKeychainEntries = [NSMutableArray<NSDictionary *> new];
 }
 
 - (void)tearDown;
 {
-    [super tearDown];
-    
     for (VALValet *const additionalValet in self.additionalValets) {
         [additionalValet removeAllObjects];
     }
+
+    for (NSDictionary *keychainEntry in self.manualKeychainEntries) {
```

Comment:
seems much simpler to do this than some of the other workarounds to reset the keychain:

- https://medium.com/@ryuichi/setup-host-app-for-keychain-tests-in-xcode-8-97222611917e
- https://stackoverflow.com/questions/34907873